jetbrains-core/src/software/aws/toolkits/jetbrains/services/clouddebug/actions/DeinstrumentResourceAction.kt

```diff
@@ -75,4 +77,14 @@ internal class DeinstrumentAction(
             .withParameters("--service")
             .withParameters(serviceName)
     }
+
+    override fun produceTelemetry(startTime: Instant, result: Result, version: String?) {
```

Comment:
These telemetry changes make it pretty difficult to see what's related to credential pr

Reply:
Master has been merged in to clean that up